src/Administration/Resources/app/administration/src/app/component/form/sw-custom-field-set-renderer/index.js

```diff
@@ -113,15 +119,15 @@ Component.register('sw-custom-field-set-renderer', {
             }
 
             // customFieldSetSelectionActive not set and parent product has no selection
-            if (this.entity.customFieldSetSelectionActive === null
-                && this.getInheritValue('customFieldSetSelectionActive') === null) {
-                return false;
-            }
-
-            return true;
+            return !(this.entity.customFieldSetSelectionActive === null
+                && this.getInheritValue('customFieldSetSelectionActive') === null);
         },
 
         visibleCustomFieldSets() {
+            // needed for recalculation only
+            // eslint-disable-next-line no-unused-expressions
+            this.refreshVisibleSets;
```

Comment:
I guess this is for reactivity reasons. But it shoulnt be needed. You can use this.$set() in line 371. That should fix it.